pages/about/index.tsx

```diff
@@ -213,6 +213,28 @@ const About: NextPage<{ teams: TeamSection[] }> = ({ teams }) => {
           ]}
         />
 
+        {currentLanguage.toString() === 'cs' || currentLanguage.toString() === 'uk' ? (
+          <>
+            <H2>{t('about_page.movapp_origin_title')}</H2>
+            <Trans
+              components={[
+                <TextLink
+                  href={currentLanguage.toString() === 'cs' ? 'https://www.movapp.cz/wiki/pribeh' : 'https://www.movapp.cz/wiki/istorija'}
+                  target="_blank"
+                  key="plakatky"
+                />,
+                <TextLink
+                  href={currentLanguage.toString() === 'cs' ? '/wiki/vyvoj-titulni-stranky' : '/wiki/zminy-na-holovnii-storintsi'}
+                  target="_blank"
+                  key="kontakt"
```

Comment:
key="kontakt" ? - tohle tu taky asi být nemá

Reply:
Ano máš pravdu, kopíroval jsem to odjinud a přehlédnul to. Teď je to spravené